src/templates/index.html

```diff
@@ -0,0 +1,65 @@
+<ovh-chatbot-icons></ovh-chatbot-icons>
+
+<div class="chatbot-main"
+        data-ng-hide="$ctrl.hidden"
```

Comment:
Indentation looks a bit off doesn't it ?

(in know it looks good in your IDE :) )